.codespellrc

```diff
@@ -1,3 +1,3 @@
 [codespell]
-ignore-words-list = upto,nd,ba,doas,fo,struc,shouldbe,iterm,lates,testof
+ignore-words-list = upto,nd,ba,bu,doas,fo,struc,shouldbe,iterm,lates,testof
```

Comment:
In the generated HTML for Bash examples. For example, from *./website/learn/unique-semantics.html*:

```
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="bu">echo</span> <span class="va">$?</span> <span class="co"># prints &quot;0&quot;</span></span>
```

Another option to whitelisting terms like "bu" is to blacklist the generated HTML files since the objectionable word does not appear in the Markdown file. In fact, I think I'll take that approach in the next commit I'm going to send for review that changes the `make lint` target to execute all lint checks rather than just `staticcheck` and `go vet`.